src/quo/components/notifications/toast/style.cljs

```diff
@@ -38,8 +38,14 @@
   {:color (colors/theme-colors colors/white colors/neutral-100 theme)})
 
 (defn icon
-  [theme]
-  {:color           (colors/theme-colors colors/white colors/neutral-100 theme)
+  [type theme]
+  {:color           (condp = type
+                      :negative (colors/resolve-color :danger theme)
+                      :positive (colors/resolve-color :success theme)
+                      (colors/theme-colors colors/white-opa-40
+                                           colors/neutral-80-opa-40
+                                           theme)
+                    )
```

Comment:
Please fix the trailing parenthesis. Also this `condp` could perfectly  be a `case`, which is simper